### PR TITLE
[testing only, do not merge] Azure memory warnings with a simple build

### PR DESCRIPTION
**Pull request by aliciaaevans**
There seems to be an increase in out of memory errors and low memory warnings on Azure. I want to see if a build with a low number of dependencies still has those warnings.

Also may make some CI changes to troubleshoot.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new test package `test` (version 0.4) to enhance testing capabilities.
	- Added a command to verify the installation of `seqtk`.

- **Documentation**
	- Included metadata for the new package, such as homepage and license information.

### CODE REVIEW

**Comment by aliciaaevans** [comment]
@coderabbitai configuration

**Comment by aliciaaevans** [comment]
@coderabbitai review

**Comment by aliciaaevans** [comment]
@coderabbitai Bioconda only supports `meta.yaml` files, not `meta.yml` files. The extension must be `yaml`. In future reviews, can you check for that?

File: recipes/test/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
+
+make CC=${CC} CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" all
```
Comment (martin-g) [inline recipes/test/build.sh:6]:
```suggestion
make -j ${CPU_COUNT} CC=${CC} CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" all
```
to make it parallel and thus use more CPU & RAM

**Pull request closed without merging**